core/storage/feedback/gae_models.py

```diff
@@ -110,19 +143,31 @@ def create(cls, exploration_id, thread_id):
     def get_by_exp_and_thread_id(cls, exploration_id, thread_id):
         """Gets the FeedbackThreadModel entry for the given ID.
 
-        Returns None if the thread is not found or is already deleted.
+        Args:
+            exploration_id: str. ID of the exploration.
+            thread_id: str. ID of the thread.
+
+        Returns:
+            FeedBackThreadModel or none if the thread is not found or is
+                already deleted.
         """
         return cls.get_by_id(cls.generate_full_thread_id(
             exploration_id, thread_id))
 
     @classmethod
     def get_threads(cls, exploration_id, limit=feconf.DEFAULT_QUERY_LIMIT):
-        """Returns a list of threads associated to the exploration, ordered by
-        their last updated field. The number of entities fetched is limited by
-        the `limit` argument to this method, whose default value is equal to the
-        the default query limit.
-
-        Does not include the deleted entries.
+        """Returns a list of threads associated to the exploration, ordered
+            by their last updated field. The number of entities fetched is
+            limited by the `limit` argument to this method, whose default
+            value is equal to the the default query limit.
+
+        Args:
+            exploration_id: str.
+            limit: int. Default query limit which is set to
```

Comment:
Yes, but explain in more detail what that means in terms of the output that is returned. Presumably it's the maximum possible number of items in the returned list?